app/src/main/java/com/nishan/moviedb/APIClient.java

```diff
@@ -0,0 +1,20 @@
+package com.nishan.moviedb;
+
+import android.content.Context;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+class APIClient {
```

Comment:
Try out OkHttpClient and configure it to log responses. It is useful for debugging. https://github.com/square/retrofit/issues/467